<ui version="4.0" >
 <class>CellTypeColorDialog</class>
 <widget class="QDialog" name="CellTypeColorDialog" >
  <property name="enabled" >
   <bool>true</bool>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>340</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>420</width>
    <height>340</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>420</width>
    <height>340</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Current color assignments</string>
  </property>
  <widget class="QWidget" name="verticalLayout" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>402</width>
     <height>320</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" >
    <item>
     <widget class="QGroupBox" name="groupBox" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Expanding" hsizetype="Preferred" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize" >
       <size>
        <width>400</width>
        <height>200</height>
       </size>
      </property>
      <property name="baseSize" >
       <size>
        <width>470</width>
        <height>200</height>
       </size>
      </property>
      <property name="title" >
       <string>Cell Type</string>
      </property>
      <widget class="QWidget" name="layoutWidget_2" >
       <property name="geometry" >
        <rect>
         <x>10</x>
         <y>20</y>
         <width>381</width>
         <height>131</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" >
        <property name="spacing" >
         <number>6</number>
        </property>
        <property name="leftMargin" >
         <number>0</number>
        </property>
        <property name="topMargin" >
         <number>0</number>
        </property>
        <property name="rightMargin" >
         <number>0</number>
        </property>
        <property name="bottomMargin" >
         <number>0</number>
        </property>
        <item>
         <widget class="QTableWidget" name="typeColorTable" >
          <property name="minimumSize" >
           <size>
            <width>245</width>
            <height>0</height>
           </size>
          </property>
          <property name="rowCount" >
           <number>3</number>
          </property>
          <property name="columnCount" >
           <number>2</number>
          </property>
          <row/>
          <row/>
          <row/>
          <column>
           <property name="text" >
            <string>Cell Type</string>
           </property>
          </column>
          <column>
           <property name="text" >
            <string>Color</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="textLabel1" >
          <property name="text" >
           <string>Click any cell 
in the column "Color" 
to change the color
for a given cell type</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_3" >
       <property name="geometry" >
        <rect>
         <x>10</x>
         <y>150</y>
         <width>381</width>
         <height>41</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" >
        <property name="spacing" >
         <number>6</number>
        </property>
        <property name="leftMargin" >
         <number>0</number>
        </property>
        <property name="topMargin" >
         <number>0</number>
        </property>
        <property name="rightMargin" >
         <number>0</number>
        </property>
        <property name="bottomMargin" >
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" >
          <property name="spacing" >
           <number>6</number>
          </property>
          <property name="leftMargin" >
           <number>0</number>
          </property>
          <property name="topMargin" >
           <number>5</number>
          </property>
          <property name="rightMargin" >
           <number>0</number>
          </property>
          <property name="bottomMargin" >
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="newCellTypeButton" >
            <property name="text" >
             <string>Add Cell Type </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="deleteCellTypeButton" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="text" >
             <string>Delete Cell Type</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="textLabel1_3" >
            <property name="text" >
             <string>Add or Delete cell type</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" >
      <item>
       <layout class="QGridLayout" >
        <property name="rightMargin" >
         <number>7</number>
        </property>
        <item row="0" column="0" >
         <widget class="QLabel" name="textLabel1_2" >
          <property name="minimumSize" >
           <size>
            <width>120</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>120</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text" >
           <string>Cell Border Color</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QLabel" name="textLabel1_2_2" >
          <property name="text" >
           <string>Contour Color</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QPushButton" name="borderColorButton" >
          <property name="minimumSize" >
           <size>
            <width>90</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>90</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text" >
           <string>Choose Color</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="QPushButton" name="contourColorButton" >
          <property name="minimumSize" >
           <size>
            <width>90</width>
            <height>0</height>
           </size>
          </property>
          <property name="text" >
           <string>Choose Color</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2" >
         <widget class="QLabel" name="borderColorLabel" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize" >
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="baseSize" >
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="autoFillBackground" >
           <bool>true</bool>
          </property>
          <property name="text" >
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="2" >
         <widget class="QLabel" name="contourColorLabel" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize" >
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="baseSize" >
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="palette" >
           <palette>
            <active>
             <colorrole role="WindowText" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>0</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Button" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>255</red>
                <green>255</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Light" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>255</red>
                <green>255</green>
                <blue>127</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Midlight" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>255</red>
                <green>255</green>
                <blue>63</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Dark" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>127</red>
                <green>127</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Mid" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>170</red>
                <green>170</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Text" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>0</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="BrightText" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="ButtonText" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>0</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Base" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Window" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>255</red>
                <green>255</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Shadow" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>0</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Highlight" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>103</red>
                <green>141</green>
                <blue>178</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="HighlightedText" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Link" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>0</red>
                <green>0</green>
                <blue>238</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="LinkVisited" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>82</red>
                <green>24</green>
                <blue>139</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="AlternateBase" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>255</red>
                <green>255</green>
                <blue>63</blue>
               </color>
              </brush>
             </colorrole>
            </active>
            <inactive>
             <colorrole role="WindowText" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>0</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Button" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>255</red>
                <green>255</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Light" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>255</red>
                <green>255</green>
                <blue>127</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Midlight" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>255</red>
                <green>255</green>
                <blue>63</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Dark" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>127</red>
                <green>127</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Mid" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>170</red>
                <green>170</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Text" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>0</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="BrightText" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="ButtonText" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>0</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Base" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Window" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>255</red>
                <green>255</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Shadow" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>0</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Highlight" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>103</red>
                <green>141</green>
                <blue>178</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="HighlightedText" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Link" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>0</red>
                <green>0</green>
                <blue>238</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="LinkVisited" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>82</red>
                <green>24</green>
                <blue>139</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="AlternateBase" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>255</red>
                <green>255</green>
                <blue>63</blue>
               </color>
              </brush>
             </colorrole>
            </inactive>
            <disabled>
             <colorrole role="WindowText" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>127</red>
                <green>127</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Button" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>255</red>
                <green>255</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Light" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>255</red>
                <green>255</green>
                <blue>127</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Midlight" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>255</red>
                <green>255</green>
                <blue>63</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Dark" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>127</red>
                <green>127</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Mid" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>170</red>
                <green>170</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Text" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>127</red>
                <green>127</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="BrightText" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="ButtonText" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>127</red>
                <green>127</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Base" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>255</red>
                <green>255</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Window" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>255</red>
                <green>255</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Shadow" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>0</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Highlight" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>86</red>
                <green>117</green>
                <blue>148</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="HighlightedText" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Link" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>0</red>
                <green>0</green>
                <blue>238</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="LinkVisited" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>82</red>
                <green>24</green>
                <blue>139</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="AlternateBase" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>255</red>
                <green>255</green>
                <blue>63</blue>
               </color>
              </brush>
             </colorrole>
            </disabled>
           </palette>
          </property>
          <property name="autoFillBackground" >
           <bool>true</bool>
          </property>
          <property name="text" >
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>165</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" >
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="okButton" >
        <property name="text" >
         <string>OK</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancelButton" >
        <property name="text" >
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>CellTypeColorDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>343</x>
     <y>304</y>
    </hint>
    <hint type="destinationlabel" >
     <x>239</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>CellTypeColorDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>429</x>
     <y>304</y>
    </hint>
    <hint type="destinationlabel" >
     <x>239</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
